derive/Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "bincode_derive"
-version = "2.0.0-rc.3" # remember to update bincode
+version = "2.0.0" # remember to update bincode
 authors = [
     "Zoey Riordan <zoey@dos.cafe>",
     "Victor Koenders <bincode@trangar.com>",
 ]
 edition = "2021"

```

Comment:
Rust 1.85 brings support for Edition2024, maybe that could be updated as well?

Reply:
We currently don't see the point of upgrading to 2024, as the rust compiler will still be able to run 2021 code. We do have the option to bump to 2024 later if we want to start adding new 2024 features